src/components/pages/Dashboard/Dashboard.js

```diff
@@ -8,7 +8,7 @@ import LandingPage from '../LandingPage/LandingPage';
 
 const Dashboard = props => {
   const { user } = useAuth0();
-  if (user.email.includes('002')) {
+  if (user.email.includes('001') || user.email.includes('002')) {
```

Comment:
Good job catching that... I wrote this conditional so sorry for not seeing that initially... Eventually we need to write more generic router, but for now this works.